bokehjs/src/coffee/polyfill.ts

```diff
@@ -0,0 +1,9 @@
+declare function require(moduleName: string): any
+
+if (typeof WeakMap !== "function") {
+  require("es6-weak-map/implement")
+}
+
+if (typeof Set !== "function") {
+  require("es6-set/implement")
+}
```

Comment:
We are currently being funded to support PhantomJS as a target for static output generation by folks who can't really use Chrome. So either we find a way to include this only when generating output with PhantomJS, or we are stuck with it for a long time.